english/coffee/cookie/bake-cookies.md

```diff
@@ -5,3 +5,8 @@ You satiate yourself with a pan
 full of freshly baked cookies.
 
 Yum.
+
+After having coffee and cookies, you would want to:
+
+[Go for a walk](walk/walk.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.